core/app/services/uffizzi_core/compose_file/parsers/services/restart_parser_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class UffizziCore::ComposeFile::Parsers::Services::RestartParserService
```

Comment:
Maybe we can rename `RestartParserService` to `RestartService`. We already have word `Parsers` in class name